### PR TITLE
add informative text for visibility settings

### PR DESCRIPTION
**Pull request by thesahindia**
### Details

Added informative text for visibiilty settings in New Room page
### Fixed Issues

$ https://github.com/Expensify/App/issues/7669

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Click "+"
2. Click "New Room"
3. Check that there is some informative text below the Visibility settings
- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/83179295/155342651-5678bb40-74ae-44c2-926a-d0e58079bf7b.mov

#### Mobile Web

https://user-images.githubusercontent.com/83179295/155342669-e066d867-64fa-4cef-b2dc-4d8003ae44a2.mov

#### Desktop

https://user-images.githubusercontent.com/83179295/155342714-bfe78e5c-1395-486e-908e-b580e90c721f.mov

#### iOS

https://user-images.githubusercontent.com/83179295/155342744-c13191f8-58e0-4275-a651-2bab6e871cac.mov

#### Android

https://user-images.githubusercontent.com/83179295/155343829-6e61faab-c191-47c8-9e17-2b9765d4fce9.mov

### CODE REVIEW

**Comment by thesahindia** [comment]
I have used the description that we already had as it was also in the mockup. Do we wanna use the description that was given in the issue?

Adding both here -

Given in the issue:-
"Restricted": "Members of your workspace can find this room"
"Private": "Only members explicitly invited can find this room"

Currently we are using:-
       "Restricted": 'People in your workspace are able to find this room using Search',
       "Private":  'Only people invited to this room are able to find it',

**Comment by puneetlath** [comment]
I prefer the existing ones. @MitchExpensify what do you think?

**Comment by rushatgabhane** [comment]
> Restricted: People in your workspace can find this room
Private: People invited to this room can find it

Asked for translations on [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1645661373276219) just in case we go with this.

**Comment by iwiznia** [comment]
Hi!
Why are we removing `using Search` from the first one?

Anyway, if we want to make that change, the only thing we would need to change in the spanish translations is to remove ` a través de "Buscar"`. The rest should remain the same, as the meaning did not really change from what it was.

**Comment by puneetlath** [comment]
I don't think the "using search" part is necessary for the description. It's just about who can find it. They don't need to know the mechanism for finding it.

**Review by rushatgabhane — CHANGES_REQUESTED** [review]

File: src/pages/workspace/WorkspaceNewRoomPage.js [hunk 1]
```diff
@@ -130,12 +131,17 @@ class WorkspaceNewRoomPage extends React.Component {
                                 onChange={this.onWorkspaceSelect}
                             />
                         </View>
-                        <Picker
-                            value={this.state.visibility}
-                            label={this.props.translate('newRoomPage.visibility')}
-                            items={visibilityOptions}
-                            onChange={visibility => this.setState({visibility})}
-                        />
+                        <View style={styles.mb2}>
```
[the remark this answers is not in the document]
Comment (thesahindia) [reply src/pages/workspace/WorkspaceNewRoomPage.js:134]: > 1. Let's update the copy

Done.

> 2. Picker alignment seems off. (`styles.w100` might help)

It's the default behaviour I guess as it's same on other places. Attaching a screenshot.
Screenshot 2022-02-28 at 10 16 51 PM

[the remark this answers is not in the document]
Comment (rushatgabhane) [reply src/pages/workspace/WorkspaceNewRoomPage.js:134]: ohh, that's weird

**Review by rushatgabhane — APPROVED** [review]
@puneetlath LGTM!

**Review by puneetlath — APPROVED** [review]

**Comment by Stutikuls** [comment]
**Issue 1 -**

**Title**- [Medium]: Chrome +Talkback: Screen reader : Focus stuck on the 'Visibility' control twice.
**Actual** - Focus lands on the Visibility control twice. first time its reading "Restricted" and 2nd time screen reader is reading "Visibility".
**Expected** - Focus should stay on the control only once and screen reader should read like "Visibility Restricted drop down button collapsed".
**WCAG failure -** 2.4.3
**Reproducible in staging?**: - Yes
**Version Number:** - v1.1.44-3
**Platforms** - Android

https://user-images.githubusercontent.com/97600950/160066638-cdccea30-e806-4d26-ba07-3d58207b9907.mp4


**Issue 2 -**

**Title**- [Medium]: Talkback+ Chrome: Screen reader : Incorrect role and state is defined for the 'Visibility' control.
**Actual** - When focus lands on the Visibility control then screen reader is reading "Restricted" and 2nd time screen reader is reading "Visibility"  and focus lands on the label, value of the control individually.
**Expected** - Role = Drop down, State = Collapsed/expanded, Value = Restricted/private. Focus should land on the control only once and when focus lands on the control then screen reader should read like "Visibility Restricted drop down button collapsed".
**WCAG failure -** 4.1.2
**Reproducible in staging?**: - Yes
**Version Number:** - v1.1.44-3
**Platforms** - Android

https://user-images.githubusercontent.com/97600950/160067552-c2958704-040e-403f-9f4f-9c1782a3354e.mp4

**Issue 3 -**

**Title**- [Medium]: Talkback+ Chrome: Screen reader :'Visibility' control drop down is not accessible using screen reader.
**Actual** -  Visibility control is not accessible using screen reader.
**Expected** - 'Visibility' control drop down should accessible using screen reader.
**WCAG failure -** 2.1.1
**Reproducible in staging?**: - Yes
**Version Number:** - v1.1.44-3
**Platforms** - Android

https://user-images.githubusercontent.com/97600950/160067266-a1f4cedd-875a-405d-ae01-c051cf0c7e69.mp4

**Pull request merged**